hack/jenkins/e2e.sh

```diff
@@ -1145,8 +1145,9 @@ if [[ "${E2E_UP,,}" == "true" || "${JENKINS_FORCE_GET_TARS:-}" =~ ^[yY]$ ]]; the
         # Otherwise, we want a completely empty directory.
         if [[ "${JENKINS_FORCE_GET_TARS:-}" =~ ^[yY]$ ]]; then
             rm -rf kubernetes*
-        elif [[ $(find . -not -path './.config*' | wc -l) != 1 ]]; then
-            echo $PWD not empty, bailing!
+        elif [[ $(find . -not -path "./.config*" -not -name "console-log.txt" \
```

Comment:
Sorry to request change after lgtm, but can you add a short comment here explaining why we expect these files to show up in a "clean" workspace?